pyproject.toml

```
@@ -0,0 +1,65 @@
+[tool.poetry]
+name = "caracal"
+version = "1.1.0"
+description = "Containerized Automated Radio Astronomy Calibration"
+authors = ["The CaraCAL Team <caracal-info@googlegroups.com>"]
+readme = "README.md"
+homepage = "https://github.com/caracal-pipeline/caracal"
+repository = "https://github.com/caracal-pipeline/caracal"
+keywords = ["Astronomy", "Visualisation", "Pipelines"]
+classifiers = [
+          "Development Status :: 5 - Production/Stable",
+          "Intended Audience :: Science/Research",
+          "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
+          "Operating System :: POSIX :: Linux",
+          "Programming Language :: Python",
+          "Topic :: Scientific/Engineering :: Astronomy"
+]
+license = "GNU GPL v2"
+packages = [
+    {include = "caracal"},
+]
+
+[tool.poetry.scripts]
+shadems = {reference = "bin/caracal", type = "file"}
```

I think this should be `caracal = ` instead of `shade-ms =`

Is this script specification style supported or from an older version of poetry? The documentation doesn't mention it: https://python-poetry.org/docs/pyproject/#scripts, but it does mention a `{callable = ` style.

The following style seems to be standard and will auto-generate a script that calls `caracal.main.main` (i.e. `bin/caracal`)

```suggestion
caracal = "caracal.main:driver"
```

Then, in `caracal/main.py`, one can do something like this:

```python
def driver():
  import sys
  main(sys.argv[1:])

def main(argv):
  ...
```